### DISCUSSION TITLE
Run without docker

### DISCUSSION THREAD

**Discussion by JxxIT** [ptrumpis/snap-camera-server#118] [Q&A] [answered]
Hi, I want to use snap-camera-server without usage of docker. Is that possible?

**Comment by ptrumpis** [comment, marked as answer]
Yes, but you need another local or remote hosting solution

[Tutorial install on CPanel shared hosting (no docker)](https://github.com/ptrumpis/snap-camera-server/discussions/80)

**Reply by JxxIT** [reply]
Thanks!
